docs/installation.rst

```diff
@@ -204,18 +204,33 @@ management command. Replace ``YOUR_USERNAME`` with your username and
     --password YOUR_PASSWORD
 
 With a password-enabled admin account, you can log into Django admin at
-http://localhost:8000/admin/login/
+http://localhost.org:8000/admin/login
 
 .. _enable-github-auth:
 
+Update the Sites section
+=======================================
+#. After logging in to the Django admin (an alternative is using the login ``test-super`` 
+   with password ``test-password``), scroll down to the Sites section. 
+
+#. Click on "Change". 
+
+#. Click on the entry that says ``localhost:8000``. 
+
+#. Change both the domain and display name from ``localhost:8000`` to ``localhost.org:8000``. 
+
+#. Click "Save".
+
+
+
 Enable GitHub/Google authentication (optional)
 =======================================
 Since Google's OAuth requires us to use a valid top-level-domain, we're going to use
 http://localhost.org:8000 as an example for every URL here.
 
 To automate setting Django up for social auth you can run
 ``docker-compose exec web ./manage.py configure_social_auth`` and follow its steps (and
-ignore the rest of this section).
+ignore the rest of this section). 
```

Comment:
Your editor doesn't seem to trim trailing spaces upon save. If you have VSCode or Atom, it should be easy to set up.

Reply:
Removed extra space-- I was editing in GitHub's UI, so that's probably why it snuck in there. I'll make sure VSCode has this setting set.